### DISCUSSION TITLE
Project creation from template

### DISCUSSION THREAD

**Discussion by fabio-percivaldi** [mia-platform/community#174] [Q&A] [answered]
Hi everyone,

i created a new project for a new company as i already did in the past and this time the configurations folder is basically empty. After doing a commit the automatically generated files are present but the default files like the pipeline yml has not been copied from the template. 

Am i missing something?

**Comment by davidebianchi** [comment]
Links are private, so it is better to not link here but only explain what it is happened. Maybe the git provider of your company has not access to this template, should we further investigate.

**Comment by frengooo** [comment]
Hi guys,
I use this thread for a related issue.
This issue is related to the topic "I had a problem during the project initialization". At the moment the user who creates a project starting from a template has different possibilities:
- the creation is OK &rarr; the first commit and the first deploy from the console work fine;
- the creation is KO &rarr; the user receives the following error on the console
![image001](https://github.com/mia-platform/community/assets/28152586/33333f69-1e08-415f-be0b-501323c0bdd2)

- the creation is OK but something goes wrong &rarr; the creation is OK but the repo does not contain all the files or the first commit/deploy fails.

For the last 2 possibilities the user is not aware of the specific problem, is there a way to discover the root cause of the problem without the need to access the project-creation service logs?

**Reply by davidebianchi** [reply, marked as answer]
Not at the moment, we should improve the error message returned by the API, so that it's clear what it missing. Because it could also be that the creation of the repo to the git provider fails

**Reply by frengooo** [reply]
Do you already have a related issue or should I open it?

**Reply by davidebianchi** [reply]
I don't remember, I think it's better if you open a new issue! Thank you

**Reply by frengooo** [reply]
I just created a new issue. Thanks for the support.
